src/utils/create-bitwise-enum-flags-class.ts

```
@@ -0,0 +1,234 @@
+      this._value = Number(initialValue);
+    }
+
+    static isValidFlagKey(flagKey: string): boolean {
+      if (!Object.prototype.hasOwnProperty.call(enumObject, flagKey))
+        return false;
+      return isNaN(Number(flagKey));
+    }
+
+    static getFlagKey(flag: NumVal | number): string {
+      for (const key in enumObject) {
+        if (!this.isValidFlagKey(key)) continue;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)